docs/cas-server-documentation/installation/Cassandra-ticket-registry.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: CAS - Cassandra Ticket Registry
+---
+
+# Cassandra Ticket Registry
+Cassandra integration is enabled by including the following dependency in the WAR overlay:
+
+```xml
+<dependency>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-cassandra-ticket-registry</artifactId>
+    <version>${cas.version}</version>
+</dependency>
+```
+
+## Configuration Considerations
+
+There are four core configuration concerns with Cassandra:
+
+1. Keyspace
+2. Ticket cleaning
+3. Object serialization mechanism
+4. Multi datacenter replication
+
+### Keyspace
+We suggest the following Keysapace definition:
+
+- String serialized ticket:
+```cql
```

Comment:
Is there an opportunity for CAS to auto-create the keyspace, optionally, similar to how JPA ticket registry works so that these can all be removed from the docs? I have not thoroughly reviewed this all, but I suspect there are certain assumptions made in the registry core about the schema that loosely match these schemas. It's best to say "CAS auto-creates everything that is needed in the schema so you don't bother with it"